x-pack/plugins/secops/public/lib/compose/testing_compose.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import 'ui/autoload/all';
+// @ts-ignore: path dynamic for kibana
+import chrome from 'ui/chrome';
+// @ts-ignore: path dynamic for kibana
+import { uiModules } from 'ui/modules';
+import uiRoutes from 'ui/routes';
+// @ts-ignore: path dynamic for kibana
+import { timezoneProvider } from 'ui/vis/lib/timezone';
+
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import ApolloClient from 'apollo-client';
+import { SchemaLink } from 'apollo-link-schema';
+import { addMockFunctionsToSchema, makeExecutableSchema } from 'graphql-tools';
+import { AppKibanaFrameworkAdapter } from '../adapters/framework/kibana_framework_adapter';
+import { AppKibanaObservableApiAdapter } from '../adapters/observable_api/kibana_observable_api';
+import { AppFrontendLibs } from '../lib';
+
+export function compose(): AppFrontendLibs {
+  const infraModule = uiModules.get('app/infa');
+  const observableApi = new AppKibanaObservableApiAdapter({
+    basePath: chrome.getBasePath(),
+    xsrfToken: chrome.getXsrfToken(),
+  });
+  const framework = new AppKibanaFrameworkAdapter(infraModule, uiRoutes, timezoneProvider);
```

Comment:
Change labeling from `infra` to `sec` or better yet remove the prefixes since we have a folder based namespace.